src/foundry/common/prosemirror/plugin.d.mts

```diff
@@ -19,5 +19,5 @@ declare abstract class ProseMirrorPlugin {
    * @param options - Additional options to pass to the plugin.
    * @virtual
    */
-  static build(schema: Schema, options?: Record<string, unknown>): Plugin;
+  static build(schema: Schema, options?: Record<string, unknown> | undefined): Plugin;
```

Comment:
Why did you choose to stick with `Record<string, unknown>` here instead of `EmptyObject`?

Reply:
This one is the parent class.  `build` has to be overridden, because this one just throws an error.  But here, you would need the flexibility to have options in order to make your own plugin.

Reply:
Alright, make it `AnyObject` then.